cdap-ui/app/directives/timeline/timeline.js

```diff
@@ -83,6 +85,7 @@ function link (scope, element) {
     scrollPinSvg = undefined;
     xAxis = undefined;
     sliderBar = undefined;
+    tooltipDiv = undefined;
```

Comment:
Do we need to explicitly define the variables as ```undefined```? isn't the default ```undefined```? Seems to be too redundant